packages/synapse-react-client/src/components/ChallengeSubmission/SubmissionDirectoryList.tsx

```diff
@@ -123,9 +129,15 @@ function SubmissionDirectoryList({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [entityType, pageSize])
 
-  const { isLoading: areEntitiesLoading, data: entities } = useGetEntities(
-    getPageHeaders(),
+  const queries = useGetEntities(getPageHeaders())
+  const entities = useMemo(
+    () =>
+      queries
+        .filter(q => q.status === 'success')
+        .map(q => q.data) as EntityItem[],
+    [queries],
   )
+  const areEntitiesLoading = queries.some(q => q.isLoading)
```

Comment:
The implementation of `useGetEntities` was changed to directly return the result of `useQueries`. Update the usage here to properly compute `entities` and `areEntitiesLoading`